engine/time/src/test/java/io/deephaven/time/calendar/CalendarInit.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2016-2023 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.time.calendar;
+
+final class CalendarInit {
+
+    static {
+        for (BusinessCalendar calendar : Calendars.calendarsFromConfiguration()) {
+            Calendars.addCalendar(calendar);
+        }
+    }
+
+    static void noop() {
```

Comment:
I would document why this method exists.  Also add a class doc about the purpose of the class.